rflx/model.py

```diff
@@ -291,13 +291,28 @@ def __repr__(self) -> str:
         return generic_repr(self.__class__.__name__, self._asdict())
 
 
-class AbstractMessage(Element):
+class AbstractMessage(Type):
+    @property
+    def size(self) -> Expr:
+        msg_size = Number(0)
+        for field in self.__fields:
+            f = self.__types[field].size
+            if isinstance(f, Number):
+                msg_size += self.__types[field].size
+            else:
+                return UNDEFINED
+
+        return msg_size
+
+    def constraints(self, name: str, proof: bool = False) -> Expr:
+        raise NotImplementedError
+
```

Comment:
I think a size property for message is not very helpful. It will only give a result for messages which only consist of static fields and that is not the usual case.

In #192 the size property and constraints function will be removed from all types except scalar types.